core/escrow/escrow.go

```diff
@@ -67,33 +68,82 @@ func NewContractRequest(configuration *config.Config, signedContracts []*escrowp
 	}, nil
 }
 
-func SubmitContractToEscrow(configuration *config.Config, request *escrowpb.EscrowContractRequest) error {
-	var conn *grpc.ClientConn
-	// TODO: Make escrow IP hidden in config too, now for testing purpose leave it here
-	conn, err := grpc.Dial("52.15.101.94:50051", grpc.WithInsecure())
-	if err != nil {
-		return err
-	}
-	defer conn.Close()
-	client := escrowpb.NewEscrowServiceClient(conn)
-	response, err := client.SubmitContracts(context.Background(), request)
-	if err != nil {
-		return err
-	}
-	if response == nil {
-		return fmt.Errorf("escrow reponse is nil")
-	}
+func SubmitContractToEscrow(configuration *config.Config, request *escrowpb.EscrowContractRequest) (
+	response *escrowpb.SignedSubmitContractResult, err error) {
+	//TODO: for testing purpose
+	address := "52.15.101.94:50051"
```

Comment:
ready from `configuration.Services.EscrowDomain` please. For testing, please change your local `~/.btfs/config`value for this field to `http://52.15.101.94:50051`

Reply:
(otherwise the testbed won't work well)

Reply:
oh, i see. that's great we can override it somewhere.